AntiGit/Git.cs

```diff
@@ -250,6 +249,7 @@ void SyncGit(ref DateTime returnOldestFile, ref bool hasSynchronized, Scan scan,
 						}
 						catch (Exception e)
 						{
+							AntiGit.WriteOutput(e.Message);
```

Comment:
SyncGit viene eseguito in continuo, si veda la linea di codice numero 30, se capita che un file non piò temporaneamente essere scritto o sovrascritto, lo sarà al prossimo giro, con il messaggio qui dentro l'utente si vedrebbe random (molto raramente) visualizzare degli errori che non comportano nulla. Probabilmente qui è opportuno mettere un avviso solo per la casistica di disco pieno (una volta).

Reply:
Exception è una eccezione generica. Andrebbe comunque messo un warning se c'è un problema di sovrascrittura perché, magari, il file è aperto e locked da un altro processo e saperlo non è un male.